x/auth/types/account.go

```diff
@@ -313,35 +312,10 @@ func (ma *ModuleAccount) UnmarshalJSON(bz []byte) error {
 	return nil
 }
 
-// AccountI is an interface used to store coins at a given address within state.
-// It presumes a notion of sequence numbers for replay protection,
-// a notion of account numbers for replay protection for previously pruned accounts,
-// and a pubkey for authentication purposes.
-//
-// Many complex conditions can be used in the concrete struct which implements AccountI.
-type AccountI interface {
```

Comment:
we should provide an alias here but mark it as deprecated

Reply:
Are we actually even willing to move given: https://github.com/cosmos/cosmos-sdk/issues/13211?

Reply:
When will that land?

Reply:
the issue proposed seems like a large refactor. anyways touched base with likhita on next steps here for leaving the interface here as a wrapper of the interface in types, then we call this one deprecated. I think the issue we can start on, but its unclear how long it will take us